account-kit/react/src/createConfig.ts

```diff
@@ -54,6 +57,25 @@ export const createConfig = (
     props.enablePopupOauth = true;
   }
 
+  const externalWalletSection = ui?.auth?.sections
+    .find((x) => x.some((y) => y.type === "external_wallets"))
+    ?.find((x) => x.type === "external_wallets") as
+    | Extract<AuthType, { type: "external_wallets" }>
+    | undefined;
+
+  if (
+    externalWalletSection?.walletConnect &&
+    !props.connectors?.some((x) => "type" in x && x.type === "walletConnect")
+  ) {
+    const walletConnectAuthConfig = externalWalletSection?.walletConnect;
+    const walletConnectParams = getWalletConnectParams(
+      walletConnectAuthConfig
+    )!;
+
+    props.connectors = props.connectors ?? [];
```

Comment:
Can be `props.connectors ??= []`.

Reply:
omg I forget about this all the time, yea will update to that